process/smartContract/builtInFunctions/keyValueStorage.go

```diff
@@ -92,6 +92,8 @@ func (k *saveKeyValueStorage) ProcessBuiltinFunction(
 			return nil, process.ErrNotEnoughGas
 		}
 
+		//key-value test point
```

Comment:
Is this added to be replaced in system tests?

Reply:
yup, it's a test point :D